pytest_selenium/pytest_selenium.py

```diff
@@ -303,6 +303,12 @@ def pytest_addoption(parser):
                   help='debug to exclude from capture',
                   default=os.getenv('SELENIUM_EXCLUDE_DEBUG'))
 
+    _auth_choices = ('none', 'token', 'hour', 'day')
```

Comment:
I'd prefer these values to be in upper case.

Reply:
Should I also make the values for `_capture_choices` upper case? @davehunt